pkg/ccl/multitenantccl/tenantcostclient/metrics.go

```diff
@@ -0,0 +1,22 @@
+package tenantcostclient
+
+import "github.com/cockroachdb/cockroach/pkg/util/metric"
+
+var (
+	metaCurrentBlocked = metric.Metadata{
+		Name:        "tenant.client.current_blocked",
```

Comment:
nit: what do you think of the name "tenant.cost_client.blocked_requests". "client" is a little under specific and "current" is implied by the fact the metric is gauge and including "request" makes it clear what is blocked.